src/dev/components/Data.tsx

```diff
@@ -14,7 +14,7 @@ export function Data() {
             postUrl: 256,
             inputText: 32,
             state: 4_096,
-            image: 256,
```

Comment:
Data URIs are 256 so probably makes sense to check the image and either set the limit conditionally.